h5py/_hl/group.py

```diff
@@ -111,6 +113,32 @@ def create_dataset(self, name, shape=None, dtype=None, data=None, **kwds):
                 self[name] = dset
             return dset
 
+    def create_virtual_dataset(self,VMlist=[], fill_value=0x1):
+        '''
+        Creates the virtual dataset from a list of virtual maps
+        '''
+        if type(VMlist) is not type([]):
+            VMlist = [VMlist]
+        
+        if len(VMlist)<1:
+            print "you have not given me any VirtualMaps!"
+            return 0
```

Comment:
I'm confused here, is this an error? A warning? What is the significance of returning 0?